go.mod

```diff
@@ -5,4 +5,5 @@ go 1.13
 require (
 	github.com/mattn/go-colorable v0.1.13
 	github.com/mattn/go-isatty v0.0.16
+	golang.org/x/sys v0.0.0-20220811171246-fbc7d0a398ab
```

Comment:
Why not `v0.5.0`?

Reply:
No good reason, maybe it was current when I opened the PR.

I accidentally deleted my branch so I would need to create a new PR in order to change the dependency version. Do you want me to do that?

Reply:
@martinlindhe could you update the dependency please? Let's use the latest and greatest.

Reply:
Updated in #186.